lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -2222,8 +2222,6 @@ def _apply_label_matcher(self, expression, topology):
             return self._exec(args)
         except Exception as e:
             logger.debug('Applying the expression failed: "{}", falling back to the original', e)
```

Comment:
Is it possible to catch a more specific exception?

Reply:
see previous comment